packages/ts/generator-typescript-plugin-backbone/src/EndpointMethodRequestBodyProcessor.ts

```diff
@@ -18,54 +18,87 @@ export type EndpointMethodRequestBody = ReadonlyDeep<OpenAPIV3.RequestBodyObject
 export type EndpointMethodRequestBodyProcessingResult = Readonly<{
   parameters: readonly ParameterDeclaration[];
   packedParameters?: ObjectLiteralExpression;
+  initParam: ts.Identifier;
 }>;
 
 export default class EndpointMethodRequestBodyProcessor {
+  readonly #defaultInitParamName = 'init';
+
   readonly #dependencies: DependencyManager;
   readonly #owner: Plugin;
   readonly #requestBody?: EndpointMethodRequestBody;
+  readonly #endpointRequestInitIdentifier: ts.Identifier;
 
   public constructor(
     requestBody: ReadonlyDeep<OpenAPIV3.ReferenceObject | OpenAPIV3.RequestBodyObject> | undefined,
     dependencies: DependencyManager,
     owner: Plugin,
+    endpointRequestInitIdentifier: ts.Identifier,
   ) {
     this.#owner = owner;
     this.#dependencies = dependencies;
     this.#requestBody = requestBody ? owner.resolver.resolve(requestBody) : undefined;
+    this.#endpointRequestInitIdentifier = endpointRequestInitIdentifier;
   }
 
   public process(): EndpointMethodRequestBodyProcessingResult {
     if (!this.#requestBody) {
       return {
-        parameters: [],
+        parameters: [
+          ts.factory.createParameterDeclaration(
+            undefined,
+            undefined,
+            undefined,
+            this.#defaultInitParamName,
+            ts.factory.createToken(ts.SyntaxKind.QuestionToken),
+            ts.factory.createTypeReferenceNode(this.#endpointRequestInitIdentifier),
+          ),
+        ],
+        packedParameters: ts.factory.createObjectLiteralExpression(),
+        initParam: ts.factory.createIdentifier(this.#defaultInitParamName),
```

Comment:
I guess, you don't need to make it unique manually:
```suggestion
        initParam: ts.factory.createUniqueIdentifier(this.#defaultInitParamName, ts.GeneratedIdentifierFlags.Optimistic),
```

I'm not completely sure about `ts.GeneratedIdentifierFlags.Optimistic` flag because it is not documented anywhere, so it probably would require a couple of tests before it satisfy our needs.

Reply:
I'm a bit puzzled about this one because I don't even get `createUniqueIdentifier` in the code completion and it is marked as not existing if I write it.

Reply:
Found: `createUniqueName`

Reply:
Problem is: it's not aware of other parameters' names, so I can make it fail quite easily

Reply:
Meh, that's what I was afraid of 😩 
BTW, that also means that my approach for using `createUniqueName` is insufficient. That's not good.

Reply:
Ok, I gonna address it in the different PR, let's have it this way for now.